src/test/java/uk/gov/ons/census/fwmt/tests/acceptance/steps/RequestSteps.java

```diff
@@ -228,17 +232,62 @@ public void theResponseContainsTheQuestionnaireTypeAndQuestionnaireIDAndRequesto
     assertEquals(requesterPhone, outcomeEvent.getPayload().getFulfillment().getContact().getTelNo());
   }
 
-  @Given("RM sends an update pause case job request with case ID {string}")
+  @Given("TM already has an existing job with case ID {string}")
+  public void tmAlreadyHasAnExistingJobWithCaseID(String caseId) throws URISyntaxException, InterruptedException {
+    queueUtils.sendToActionFieldQueue(receivedRMMessage);
+
+    Thread.sleep(1000);
```

Comment:
Can you remove the Thread.sleeps(), weve had very bad experiences when using them. You end up setting all the sleeps to the 'worst case scenario' length and tests begin to take ages. Each stage in process fires off events to a Q. 

For this particular instance you want to know when JobService has sent a message to TM before you check TM (Mock) for the message, if you go to JobServiceImpl.convertAndSendUpdate() you can see after calling COMET it sends a COMET_UPDATE_ACK event. you can listen to this by.
    boolean hasBeenTriggered = gatewayEventMonitor.hasEventTriggered(caseId, COMET_UPDATE_ACK, 10000L);
instead of the the sleep.

You can look at this code, but essentially it  checks every few milliseconds for the message up to, in this instance (2nd Parameter) 10000 milliseconds.